### ISSUE TITLE
bug report 

### ISSUE THREAD

**Issue by weir12** [automl/auto-sklearn#1072]
Hi:
The following error report occurs when I use AutoSklearn2Classifier:
```
~/envs/nn/lib/python3.7/site-packages/autosklearn/experimental/askl2.py in fit(self, X, y, X_test, y_test, metric, feat_type, dataset_name)
    331             dataset_name=None):
    332 
--> 333         with open(selector_file, 'rb') as fh:
    334             selector = pickle.load(fh)
    335 

IsADirectoryError: [Errno 21] Is a directory: '/home/weir12/.cache'
```
I guess there is a problem with the following code, after my review.

https://github.com/automl/auto-sklearn/blob/18d26654f7de43a67fb4eedb44ba3e02e506bbd1/autosklearn/experimental/askl2.py#L30-L36

It should be replaced by

```
selector_file = pathlib.Path(
    os.environ.get(
        'XDG_CACHE_HOME'),
        '~/.cache/auto-sklearn/askl2_selector_%s_%s_%s.pkl'
        % (autosklearn_version, sklearn_version, training_data_hash),
    ).expanduser()
```

Otherwise, the file path of selector does not seem to be generated correctly.
Thanks!

**Comment by mfeurer** [comment]
Thank you very much for reporting this. @franchuterivera will have a look at this. However, it might rather be:
```python
selector_file = pathlib.Path(os.environ.get('XDG_CACHE_HOME', pathlib.Path('~') / '.cache')
selector_file = selector_file / 'auto-sklearn' / (askl2_selector_%s_%s_%s.pkl' % (autosklearn_version, sklearn_version, training_data_hash))
selector_file = selector_file.expanduser()
```

**Comment by mfeurer** [comment]
Fixed via #1073. Will be available in the next release.
